### ISSUE TITLE
Update field with yii2 money makes value 100 times less...

### ISSUE THREAD

**Issue by Marcuzzz** [kartik-v/yii2-money#23]
I've trouble with updating a money value with the money extension.
It is saved correctly on create, also to the database but If I go and do an update, the value €2000,- is shown as: €20,00 instead of €2000,- and saved to the database as €20,00 
How to correct this behaviour?

**Comment by ilbassa** [comment]
Maybe there is a problem with the decimal separator?
I have resolved adding these lines in renderInput()
`$this->value=str_replace(',', '.', $this->value);`
`$this->options=array_merge($this->options,['value'=>$this->value]);`

**Comment by Marcuzzz** [comment]
Yes could be. 
What do you mean by renderInput...
I don't have "$this" I have a $model...

```
$this->value=str_replace(',', '.', $this->value);
$this->options=array_merge($this->options,['value'=>$this->value]);


<?= $form->field($model, 'PriceBuyout')->widget(MaskMoney::classname(), [
    'pluginOptions' => [
        'prefix' => '€ ',
        'suffix' => ' ,-',
        'allowNegative' => false
    ]
    ]);
 ?>

```

**Comment by ilbassa** [comment]
I changed the core file MaskMoney.php which has renderInput().

**Comment by Marcuzzz** [comment]
if noticed when I comment out : 
`
// suffix' => ' ,-', 
`
It does work a expected...

**Comment by kartik-v** [comment]
Resolved via [updates & enhancements to release v1.2.2](https://github.com/kartik-v/yii2-money/commit/6c18c47b6561e52ecd4cff9d2e0c3a0e974c236f). Duplicate to #26, #22, #16, #9.
